doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -0,0 +1,103 @@
+.. rn:: 8.0.16-7
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+
+|Percona Server| |release| is now the current GA release in the 8.0
+series. All of |Percona|’s software is open-source and free.
+
+Percona Server for MySQL 8.0 includes all the `features available in MySQL 8.0
+Community Edition
+<https://dev.mysql.com/doc/refman/8.0/en/mysql-nutshell.html>`__ in addition to
+enterprise-grade features developed by Percona.  For a list of highlighted
+features from both MySQL 8.0 and Percona Server for MySQL 8.0, please see the
+`GA release announcement
+<https://www.percona.com/blog/2018/12/21/announcing-general-availability-of-percona-server-for-mysql-8-0/>`__.
+
+.. note::
+
+   If you are upgrading from 5.7 to 8.0, please ensure that you read the
+   `upgrade guide
+   <https://www.percona.com/doc/percona-server/8.0/upgrading_guide.html>`__ and
+   the document `Changed in Percona Server for MySQL 8.0
+   <https://www.percona.com/doc/percona-server/8.0/changed_in_version.html>`__.
+
+New Features
+================================================================================
+
+- Key rotation for the redo log has been redesigned to use the lsn and block size to calculate the header number.
```

Comment:
"LSN" is an acronym, should be uppercased
JIRA link?
The description sounds like an implementation detail: what does it, if anything, mean for the end user?

Reply:
updated "LSN" and rewrote the description.